x/oracle/types/params.go

```diff
@@ -13,7 +13,7 @@ import (
 const (
 	// Each value below is the default value for each parameter when generating the default
 	// genesis file. See comments in types.proto for explanation for each parameter.
-	DefaultMaxRawRequestCount      = uint64(12)
+	DefaultMaxRawRequestCount      = uint64(16)
```

Comment:
Please change `DefaultPerValidatorRequestGas` to 0 (mainnet uses 0 for now)

Reply:
updated.